sql/item_jsonfunc.h

```diff
@@ -525,9 +525,9 @@ class Item_func_json_insert: public Item_json_str_multipath
   {
     static LEX_CSTRING json_set=    {STRING_WITH_LEN("json_set") };
     static LEX_CSTRING json_insert= {STRING_WITH_LEN("json_insert") };
-    static LEX_CSTRING json_update= {STRING_WITH_LEN("json_update") };
+    static LEX_CSTRING json_replace= {STRING_WITH_LEN("json_replace") };
```

Comment:
I think the `json_update` is a typo here, 'cause there is no function named `json_update` in [MariaDB JSON functions](https://mariadb.com/kb/en/json-functions/).

Reply:
Possibly. There is a related MDEV issue: https://jira.mariadb.org/browse/MDEV-24541